lib/api/v3/utilities/path_helper.rb

```diff
@@ -114,6 +114,10 @@ def self.relation(id)
             "#{root}/relations/#{id}"
           end
 
+          def self.revision(id)
+            "#{root}/revisions/#{id}"
+          end
```

Comment:
spec me ;-)

Hint: There are some older path helper specs that use regexes for unknown reasons and newer ones that do simple string comparisons...

In case you are uncertain, I would suggest using the simple string comparisons for readability reasons ^^